source/_components/wirelesstag.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "WirelessTag"
+description: "Instructions on how to integrate your Wireless Tags sensors within Home Assistant."
+date: 2018-03-26 21:32
+comments: false
+sidebar: true
+sharing: true
+footer: true
+logo: wirelesstag.png
+ha_category: Hub
+ha_iot_class: "Local Push"
+---
```

Comment:
`ha_version` is missing.

Reply:
@frenck thanks!  same here - what version would you like to me to put here? as I'm not aware to what version  this contribution will go finally ;) so i left no version intentionally...